app/controllers/application_controller.rb

```diff
@@ -90,10 +91,4 @@ def prompt_to_set_up_2fa
   def prompt_to_enter_otp
     redirect_to user_two_factor_authentication_url
   end
-
-  def track_get_requests
```

Comment:
Before, Analytics events and Rails logs were 2 separate entities. The point of the `track_get_requests` method was to include page views as part of the data we send to a third-party analytics service, since by default most events we track via calls to `analytics.track_event` are for POST requests only.

Our Rails logs have always been free of PII and other sensitive info (except for IP address) because we filter them out [here](https://github.com/18F/identity-idp/blob/master/config/initializers/filter_parameter_logging.rb) and [here](https://github.com/18F/identity-idp/blob/master/config/initializers/active_job_logger_patch.rb).

Our Analytics data is also free of PII because we do not include it. We use the user's `uuid` to identify them.

This PR consolidates analytics events and Rails logs into one place, such that all requests can be tracked in one place. Since we're not sending data to a third party service anymore (at least for now), we can use the Rails logs to contain all the data in JSON format, which can then be consumed by ELK. Since the Rails logs already contain GET requests, we don't need to generate them again as "events" via the `track_get_requests` method.

Does that make sense?